tv_apps/app-deck/bower_components/smart-modal-dialog/base-modal-dialog.js

```diff
@@ -53,8 +53,8 @@
   proto._setL10n = function(element, l10n) {
     if ((typeof l10n) === 'string') {
       element.setAttribute('data-l10n-id', l10n);
-    } else if (navigator.mozL10n) {
-      navigator.mozL10n.setAttributes(element, l10n.id, l10n.args);
```

Comment:
It's ok to just land it as is, but we need to update it to `fxos-components` too. You can open a bug for it.
`smart-modal-dialog` is now renamed to `fxos-tv-modal-dialog`. And it should replace smart-modal-dialog recently.
https://github.com/fxos-components/fxos-tv-modal-dialog